include/zephyr/net/wifi_mgmt.h

```diff
@@ -367,6 +367,10 @@ struct wifi_connect_req_params {
 
 /** Wi-Fi connect result codes. To be overlaid on top of \ref wifi_status
  * in the connect result event for detailed status.
+ * The Possible reasons for 4 way handshake failure are as follows.
```

Comment:
Shouldn't this note be for `WIFI_STATUS_CONN_WRONG_PASSWORD`? Where we can add that this is just a best guess.

Reply:
Yes it's for WIFI_STATUS_CONN_WRONG_PASSWORD, I will move note to that declaration.